app/views/users/new.html.erb

```diff
@@ -1,7 +1,14 @@
-<div>
+<div id='user_create_page'>
   <% if current_user && current_user.admin %>
-    <%= react_component 'CreateUser', { csrf: form_authenticity_token, user: @user } %>
+    <%= javascript_tag do %>
+      react_component('CreateUser', {
+        user: JSON.parse('<%= raw escape_json(@user) %>'),
```

Comment:
Have you tried the json_escape function Trevor suggested? Did it work? Then you shouldn't need the custom function escape_json anymore

Reply:
Oh I have tried it, it escaped the json using a single backslash, so I update the custom method we created to use the json_escape method trevor suggested, but to replace single backslash with double instead